src/app/Joyride/CryostatJoyride.tsx

```diff
@@ -0,0 +1,277 @@
+ *
+ * (a) the Software, and
+ * (b) any piece of software and/or hardware listed in the lrgrwrks.txt file if
+ * one is included with the Software (each a "Larger Work" to which the Software
+ * is contributed by such licensors),
+ *
+ * without restriction, including without limitation the rights to copy, create
+ * derivative works of, display, perform, and distribute the Software and make,
+ * use, sell, offer for sale, import, export, have made, and have sold the
+ * Software and the Larger Work(s), and to sublicense the foregoing rights on
+ * either these or other terms.
+ *
+ * This license is subject to the following condition:
+ * The above copyright notice and either this complete permission notice or at
+ * a minimum a reference to the UPL must be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+import cryostatLogo from '@app/assets/cryostat_logo_vert_rgb_default.svg';
+import CustomTooltip from '@app/Joyride/CustomTooltip';
+import { useJoyride } from '@app/Joyride/JoyrideProvider';
+import React from 'react';
+import ReactJoyride, { CallBackProps, ACTIONS, EVENTS, STATUS } from 'react-joyride';
+interface CryostatJoyrideProps {
+  children: React.ReactNode;
+}
+
+const CryostatJoyride: React.FC<CryostatJoyrideProps> = (props) => {
+  const {
+    setState,
+    state: { run, stepIndex, steps },
+    isNavBarOpen,
+  } = useJoyride();
+
+  React.useEffect(() => {
+    setState({
+      steps: [
+        {
+          content: (
+            <div>
+              <p>
+                <strong>Cryostat</strong> is a cloud-based profiling application for managing JFR recordings in
+                containerized Java environments.
+              </p>
+              <br />
+              <p>
+                There are many other features that <strong>Cryostat</strong> provides, such as the ability to download
+                recordings, generate reports, and more.
+              </p>
+            </div>
+          ),
+          placement: 'center',
+          title: (
+            <div style={{ padding: '1em' }}>
+              <img src={cryostatLogo} alt="Cryostat Logo" style={{ paddingBottom: '1em' }} height={300} width={300} />
+              <h1>
+                Welcome to <strong>Cryostat</strong>!
+              </h1>
+            </div>
+          ),
+          target: 'body',
+          disableBeacon: true,
+        },
+        {
+          content: <p>Open the navigation bar!</p>,
```

Comment:
Not sure if I did some thing wrong but I am seeing this:

![Screenshot from 2023-03-28 01-29-09](https://user-images.githubusercontent.com/68053619/228137517-c0619390-21a2-45d8-b2a9-d150a5055a10.png)

Reply:
Yes, it should... is there a way to replicate this?

Reply:
For me, just resize the window till the nav close, and start guided tour :))